site/_includes/sidebar.html

```diff
@@ -40,7 +40,8 @@ <h1>
 <p>
   <!-- jekyll-polyglot will process ferh= into href= through the static_href liquid block tag without relativizing the url; useful for making language navigation switchers  -->
   {% for tongue in site.languages %}
-<a {% if tongue == site.active_lang %}style="font-weight: bold;"{% endif %} {% static_href %}href="{% if tongue == site.default_lang %}{{site.baseurl}}{{page.url}}{% else %}{{site.baseurl}}/{{ tongue }}{{page.url}}{% endif %}"{% endstatic_href %} >{{ tongue }}</a> |
+{% if tongue != site.active_lang %}{{site.langsep}}{% endif %}
+<a {% if tongue == site.active_lang %}style="font-weight: bold;"{% endif %} {% static_href %}href="{% if tongue == site.default_lang %}{{site.baseurl}}{{page.url}}{% else %}{{site.baseurl}}/{{ tongue }}{{page.url}}{% endif %}"{% endstatic_href %} >{{ tongue }}</a> 
```

Comment:
Good attempt! and I like your style of inverting the logic and making a `langsep` constant, but it fails to render the `langsep` ahead of when selecting an active language that's not the first.

<img width="316" alt="Screenshot 2023-03-22 at 5 11 38 PM" src="https://user-images.githubusercontent.com/3855429/227060820-dabaf127-a288-4a51-b351-d094c6bb1a51.png">


But this was a solid attempt, and you made me take another look at the [liquid documentation,](https://shopify.github.io/liquid/tags/iteration/#forloop-properties) and I stumbled on some good clues that puts together a winning solution.

<img width="293" alt="Screenshot 2023-03-22 at 5 20 31 PM" src="https://user-images.githubusercontent.com/3855429/227060831-f9cb8217-e4a9-4e2d-b810-a7cb6054511f.png">

Thanks for your help with this one! I'll redeploy the website after a few more css updates.

```suggestion
<a {% if tongue == site.active_lang %}style="font-weight: bold;"{% endif %} {% static_href %}href="{% if tongue == site.default_lang %}{{site.baseurl}}{{page.url}}{% else %}{{site.baseurl}}/{{ tongue }}{{page.url}}{% endif %}"{% endstatic_href %} >{{ tongue }}</a>{%- if forloop.last == false -%}{{" "}}{{ site.langsep }}{%- endif -%}
```

Reply:
Thanks. 

I always forget forloop also offers first and last, whichvare also handy here.